site/learn/expert-advice/oauth/what-is-oauth.md

```diff
@@ -9,29 +9,32 @@ date: 2022-01-27
+Instead of passing authentication data between customers and service providers, an OAuth token is provided. Therefore, developers get access to end-user data in a more secure manner. In this article, you’ll learn how OAuth works, how versions 1 and 2 differ, and why you should use OAuth2 instead of your own authorization solution. 
 
 ## Why OAuth
 
-OAuth in general was designed as a reaction to the [direct authentication pattern](http://guides.brucejmack.net/SOA-Patterns/WSSP/1.3DirectvsBrokeredAuthDoc.htm), which features applications requesting usernames and passwords when accessing user-specific data at other services. When utilizing OAuth2, the user still provides this information, but they log into one platform and use tokens generated by that platform to grant access to data and perform actions in one or more other applications.
+OAuth in general was designed as a reaction to the [direct authentication pattern](http://guides.brucejmack.net/SOA-Patterns/WSSP/1.3DirectvsBrokeredAuthDoc.htm), which features applications requesting usernames and passwords when accessing user-specific data at other services. When utilizing OAuth 2.0, the user still provides this information, but they log into one platform and use tokens generated by that platform to grant access to data and perform actions in one or more other applications.
 
-Before OAuth was prevalent, websites would ask users to enter a username and password into a form, for example the credentials for a Gmail account, to gain access to Gmail data. Users realized they didn’t want to grant full access to any old service. They preferred limited access instead. A user might want a third party service to access their email account, but only have rights to read contacts, not send email or add contacts.
+Before OAuth authentication was prevalent, websites would ask users to enter a username and password into a form, for example the credentials for a Gmail account, to gain access to Gmail data. Users realized they didn’t want to grant full access to any old service. They preferred limited access instead. A user might want a third party service to access their email account, but only have rights to read contacts, not send email or add contacts.
 
-This article will focus on OAuth2, the second version of OAuth, though below we do cover differences between the versions.
+This article will focus on OAuth 2.0, the second version of OAuth, though below we do cover differences between the versions.
+
+With the help of OAuth 2.0, a token, which has a limited lifetime, provides that desired limited access. Connecting multiple applications is easier for users. It is also more secure since user credentials are not shared. Auth is also easier for developers to manage because they only need to integrate OAuth 2.0 in their application instead of having their own database to store users' sensitive information.
```

Comment:
```suggestion
When using OAuth 2.0, a token, which has a limited lifetime, provides that desired limited access. Connecting multiple applications is easier for users. It is also more secure since user credentials are not shared. Auth is also easier for developers to manage because they only need to integrate OAuth 2.0 in their application instead of having their own database to store users' sensitive information.
```